.woodpecker.yml

```diff
@@ -61,9 +61,9 @@ steps:
       - event: pull_request
 
   sql_fmt:
-    image: alpine:3
+    image: *rust_image
```

Comment:
Had to change this to have `xargs -L` available.

Reply:
Weird that alpine doesn't support it, oh well.